contracts/minters/vending-minter-wl-flex/src/contract.rs

```diff
@@ -730,9 +730,15 @@ fn _execute_mint(
         .add_attribute("sender", info.sender)
         .add_attribute("recipient", recipient_addr)
         .add_attribute("token_id", mintable_token_mapping.token_id.to_string())
-        .add_attribute("network_fee", network_fee)
-        .add_attribute("mint_price", mint_price.amount)
-        .add_attribute("seller_amount", seller_amount))
+        .add_attribute(
+            "network_fee",
+            coin(network_fee.u128(), mint_price.clone().denom).to_string(),
+        )
+        .add_attribute("mint_price", mint_price.to_string())
+        .add_attribute(
+            "seller_amount",
+            coin(seller_amount.u128(), mint_price.denom).to_string(),
```

Comment:
same here no need to do `coin(..)` again

Reply:
Same for `seller_amount`, it's a Uint128.